src/backend/logging/log_manager.h

```diff
@@ -84,40 +122,110 @@ class LogManager {
   // Whether to enable or disable synchronous commit ?
   void SetSyncCommit(bool sync_commit) { syncronization_commit = sync_commit; }
 
+  // get the status of sychronus commit
   bool GetSyncCommit(void) const { return syncronization_commit; }
 
+  // returns true if a frontend logger is active
```

Comment:
In this file, some short functions are inlined (like SetSyncCommit, GetSyncCommit) while some are not (like SetLogFileName, SetGlobalMaxFlushedCommitId). Some functions that do not modify the class are not declared as const (like GetLoggingStatus, GetLogFileName) while others are (like GetSyncCommit).
Better to unify these.
.